tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -32,6 +35,24 @@ const DayComponent = ({ date, state, marking, navigation }) => {
 }
 
 export const Calendar = ({navigation}) => {
+    const [calendarData, setCalendarData] = useState({});
+
+    //updates the data when you come back to the calendar screen
+    useEffect(() => {
+        const unsubscribe = navigation.addListener('focus', () => {
+            getCalendarData();
+        });
+        return unsubscribe;
+    }, [navigation]);
+
+    const getCalendarData = async () => {
+        // will need to choose the correct year depending on which year the user is looking at
+        const data = await getCalendarByYear(2022);
```

Comment:
I think u can get current year from a listener on the calendar? Edward did it for https://github.com/uoftblueprint/the-period-purse/pull/108

Reply:
I will ask edward about that, but for now we can just merge since we have no time lol.

Reply:
I'm ok with leaving it for later, but Roy I'll assign you to go back in and make changes then.

Reply:
yup that sounds good!